README.md

```diff
@@ -29,10 +29,24 @@ Supports Node v8.3+ and modern browsers.
 Spectral can be run via the command-line:
 
 ```bash
-$ spectral lint petstore.yaml
+spectral lint petstore.yaml
+```
+
+Other options include:
 
+```bash
+  -e, --encoding=encoding      [default: utf8] text encoding to use
+  -f, --format=json|stylish    [default: stylish] formatter to use for outputting results
+  -h, --help                   show CLI help
+  -m, --maxResults=maxResults  [default: all] maximum results to show
+  -o, --output=output          output to a file instead of stdout
+  -v, --verbose                increase verbosity
 ```
 
+> Note: The Spectral CLI supports both YAML and JSON. 
+
+Currently, the CLI supports validation of OpenAPI 2 and 3 documents and lints them based on our default ruleset. If you want to build and run custom rulesets, see [Customization](#Customization). 
```

Comment:
This kinda infers the CLI will let you then use these custom rule sets. Can we mention they’re upcoming?

Reply:
good point, updating it now....